### PR TITLE
Check for too recent of pandoc version

### PR DESCRIPTION
**Pull request by hfuhrhurr**
In researching how I can contribute to #610 at the PyCon Sprints, I noticed 2 of the 9 tests in unmodified `test_markdown.py` failed.

@ivanov suggested I make a trivial change and submit a PR so the code would run through Travis to see if the failures are replicated.

### CODE REVIEW

**Comment by hfuhrhurr** [comment]
My local tests were failing because my version of pandoc was too recent (2.2.1).  Using 1.19.1 locally was met with success.  The PR now incorporates logic to notify a user of an unsupported version of pandoc (if it's 2.0.0 or newer).  This logic won't cause a failure; it will simply inform the user an unsupported version of pandoc is being used.

@ivanov

File: nbconvert/filters/tests/test_markdown.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# trivial line added to run travis on this file
+# because, locally, 2 of the 9 tests fail
```
Comment (ivanov) [inline nbconvert/filters/tests/test_markdown.py]: Could we take this out?

**Comment by ivanov** [comment]
Thanks for your contribution @hfuhrhurr ! Merging.

**Comment by mpacer** [comment]
Btw just as a heads up, this broke our docs by introducing an API incompatibility (`max_ver` shouldn't be a positional argument, but a keyword argument).

**Pull request merged**